lib/documents/schemas/dfid_research_outputs.json

```diff
@@ -0,0 +1,835 @@
+        },
+        {
+          "value": "SD",
+          "label": "Sudan"
+        },
+        {
+          "value": "SR",
+          "label": "Suriname"
+        },
+        {
+          "value": "SZ",
+          "label": "Swaziland"
+        },
+        {
+          "value": "SE",
+          "label": "Sweden"
+        },
+        {
+          "value": "CH",
+          "label": "Switzerland"
+        },
+        {
+          "value": "SY",
+          "label": "Syria"
+        },
+        {
+          "value": "TJ",
+          "label": "Tajikistan"
+        },
+        {
+          "value": "TZ",
+          "label": "Tanzania"
+        },
+        {
+          "value": "TH",
+          "label": "Thailand"
+        },
+        {
+          "value": "BS",
+          "label": "The Bahamas"
+        },
+        {
+          "value": "GM",
+          "label": "The Gambia"
+        },
+        {
+          "value": "TG",
+          "label": "Togo"
+        },
+        {
+          "value": "TO",
+          "label": "Tonga"
+        },
+        {
+          "value": "TT",
+          "label": "Trinidad and Tobago"
+        },
+        {
+          "value": "TN",
+          "label": "Tunisia"
+        },
+        {
+          "value": "TR",
+          "label": "Turkey"
+        },
+        {
+          "value": "TM",
+          "label": "Turkmenistan"
+        },
+        {
+          "value": "TV",
+          "label": "Tuvalu"
+        },
+        {
+          "value": "SU",
+          "label": "USSR"
+        },
+        {
+          "value": "UG",
+          "label": "Uganda"
+        },
+        {
+          "value": "UA",
+          "label": "Ukraine"
+        },
+        {
+          "value": "AE",
+          "label": "United Arab Emirates"
+        },
+        {
+          "value": "GB",
+          "label": "United Kingdom"
+        },
+        {
+          "value": "US",
+          "label": "United States"
+        },
+        {
+          "value": "UY",
+          "label": "Uruguay"
+        },
+        {
+          "value": "UZ",
+          "label": "Uzbekistan"
+        },
+        {
+          "value": "VU",
+          "label": "Vanuatu"
+        },
+        {
+          "value": "VA",
+          "label": "Vatican City"
+        },
+        {
+          "value": "VE",
+          "label": "Venezuela"
+        },
+        {
+          "value": "VN",
+          "label": "Vietnam"
+        },
+        {
+          "value": "YE",
+          "label": "Yemen"
+        },
+        {
+          "value": "YU",
+          "label": "Yugoslavia"
+        },
+        {
+          "value": "ZM",
+          "label": "Zambia"
+        },
+        {
+          "value": "ZW",
+          "label": "Zimbabwe"
+        }
+      ]
+    },
+    {
+      "key": "published_date",
+      "name": "Published",
+      "short_name": "Published",
+      "type": "date",
+      "display_as_result_metadata": true,
+      "filterable": false
+    }
+  ]
+}
```

Comment:
Oh, bugger, that's as `rake patch:countries` wrote it. I guess I'll have to add it to the `Patch` classes, but make a small commit here for now.